assets/chat/js/menus/ChatUserInfoMenu.js

```diff
@@ -69,6 +49,16 @@ export default class ChatUserInfoMenu extends ChatMenu {
     });
   }
 
+  showUser(e, user, userlist = false) {
+    this.clickedNick = user.data('username');
+
+    this.setActionsVisibility();
+    this.addContent(user, userlist);
+
+    super.position(e);
```

Comment:
Is there a reason you call `super.position(e)` rather than `this.position(e)`? I only ever see `super` when calling a parent class' implementation of a method you're overriding, so this confused me. I noticed there are other seemingly inappropriate uses of `super` throughout this class, so maybe you're just following the convention.

Would you mind changing this to `this.position(e)`? And even though it's a little outside the scope of this PR, you can change other inappropriate uses of `super` to `this` as well.